worker/dbaccessor/package_test.go

```diff
@@ -1,14 +0,0 @@
-// Copyright 2021 Canonical Ltd.
-// Licensed under the AGPLv3, see LICENCE file for details.
-
-package dbaccessor
-
-import (
-	"testing"
-
-	gc "gopkg.in/check.v1"
-)
-
-func TestPackage(t *testing.T) {
-	gc.TestingT(t)
-}
```

Comment:
We can still just keep this file, even if the tests don't exist _yet_

Reply:
I'd rather we bring it back when we have some actual tests here.